### PR TITLE
doc: fix typos

### PR DESCRIPTION
**Pull request by zhongyimao**
update the discarded content in the document and fix some spelling errors.

### CODE REVIEW

**Comment by tchaikov** [comment]
@zhongyimao could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. we don't use "cleanup" to identify the subcomponent.

**Comment by zhongyimao** [comment]
@tchaikov Sorry for the terrible descriptions and thanks for the detailed clarification, just reference 
https://github.com/ceph/ceph/commit/33c8a43d6d3a2688ac1cfdc14695eabcb90e0c58
,now updated it, please review.

File: doc/rbd/rados-rbd-cmds.rst [hunk 1]
```diff
@@ -182,20 +182,20 @@ For example::
...
 .. note::
 
-  * You can move an image to the trash even it has shapshot(s) or actively 
+  * You can move an image to the trash even it has snapshot(s) or actively 
     in-use by clones, but can not be removed from trash.
 
-  * You can use *--delay* to set the defer time (default is 0), and if its 
-    deferment time has not expired, it can not be removed unless you use 
-    force.
+  * You can use *--expires-at* to set the defer time (default is now), and
```
Comment (tchaikov) [inline doc/rbd/rados-rbd-cmds.rst]: i think the `delay = 0` is more accurate.

Comment (tchaikov) [reply doc/rbd/rados-rbd-cmds.rst]: @zhongyimao i mean, a number "0" instead of "now" is better in this context. so i'd suggest keep as it is.

Comment (tchaikov) [reply doc/rbd/rados-rbd-cmds.rst]: @zhongyimao sorry, i was wrong. see https://github.com/ceph/ceph/commit/b9e4aa94b77ab9fca803df330a8e5f31e8de0f00#diff-b6fb76bfc588ba6ebd1506f4cd6403efR1584 . could you quote "now" like ``` ``now`` ``` ?

Comment (zhongyimao) [reply doc/rbd/rados-rbd-cmds.rst]: Sorry, I'm wondering if the "now"  you said is pointed https://github.com/ceph/ceph/commit/b9e4aa94b77ab9fca803df330a8e5f31e8de0f00#diff-b6fb76bfc588ba6ebd1506f4cd6403efR1584 or `+  * You can use *--expires-at* to set the defer time (default is now)`?   thank you.

Comment (zhongyimao) [reply doc/rbd/rados-rbd-cmds.rst]: ok, I got it, thanks.

**Comment by tchaikov** [comment]
jenkins render docs

**Pull request merged**